pkgs/os-specific/linux/r8125/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
       A kernel module for Realtek 8125 2.5G network cards.
     '';
     # r8125 has been integrated into the kernel as of v5.9.1
-    broken = lib.versionAtLeast kernel.version "5.9.1";
+    #broken = lib.versionAtLeast kernel.version "5.9.1";
```

Comment:
What's up with this? If it is false, it should be removed

Reply:
Thanks for the information